src/nls/messages_ja.json

```
@@ -1,304 +1,304 @@
 {
   "ja": {
-    "carbon.listBoxMenuIcon.close.menu": "",
-    "carbon.listBoxMenuIcon.open.menu": "",
-    "carbon.listBoxSelection.clear.all": "",
-    "carbon.listBoxSelection.clear.selection": "",
-    "carbon.table.all.collapse": "",
-    "carbon.table.all.expand": "",
-    "carbon.table.all.select": "行をすべて選択",
-    "carbon.table.all.unselect": "行をすべて選択解除",
+    "carbon.listBoxMenuIcon.close.menu": "メニューを閉じる",
+    "carbon.listBoxMenuIcon.open.menu": "メニューを開く",
+    "carbon.listBoxSelection.clear.all": "選択されたすべての項目をクリア",
+    "carbon.listBoxSelection.clear.selection": "選択された項目をクリア",
+    "carbon.table.all.collapse": "すべての行を折りたたむ",
+    "carbon.table.all.expand": "すべての行を展開",
+    "carbon.table.all.select": "すべての行を選択",
+    "carbon.table.all.unselect": "すべての行を選択解除",
     "carbon.table.batch.cancel": "キャンセル",
-    "carbon.table.batch.item.selected": "{totalSelected, plural, =0 {0 items} one {1 item}} が選択されました",
-    "carbon.table.batch.items.selected": "{totalSelected, plural, other {# items}} が選択されました",
-    "carbon.table.row.collapse": "",
-    "carbon.table.row.expand": "",
-    "carbon.table.row.select": "行の選択",
-    "carbon.table.row.unselect": "行の選択解除",
-    "dashboard.about.error": "",
-    "dashboard.about.isOpenShift": "",
-    "dashboard.about.isReadOnly": "",
-    "dashboard.about.logoutURL": "",
-    "dashboard.about.missingProperties": "",
-    "dashboard.about.title": "",
-    "dashboard.about.true": "",
-    "dashboard.about.version": "",
-    "dashboard.accessTokenField.accessToken": "",
-    "dashboard.accessTokenField.accessTokenRequired": "",
-    "dashboard.accessTokenField.hidePasswordLabel": "",
-    "dashboard.accessTokenField.showPasswordLabel": "",
-    "dashboard.actions.createButton": "",
-    "dashboard.actions.deleteButton": "",
-    "dashboard.app.loadingConfigError": "",
+    "carbon.table.batch.item.selected": "{totalSelected, plural, =0 {0 項目} one {1 項目}}が選択されました",
+    "carbon.table.batch.items.selected": "{totalSelected, plural, other {# 項目}}が選択されました",
```

can the plural syntax be removed from these if it's not needed?

```suggestion
    "carbon.table.batch.item.selected": "項目が選択されました",
    "carbon.table.batch.items.selected": "項目が選択されました",
```